### PR TITLE
Add recipe PATCH visibility endpoint.

### PR DESCRIPTION
**Pull request by SychovOleh**
Fix usage visibility in recipe.

### CODE REVIEW

File: source/includes/_recipes.md [hunk 1]
```diff
@@ -109,7 +110,6 @@ curl -X POST {{ api_base_url }}/v2/recipes \
     "data": {
       "type": "recipe",
       "attributes": {
-        "visibility": "workspace",
```
Comment (HannaTrotsenko) [inline source/includes/_recipes.md]: default "visibility" returns after POST request

File: source/includes/_recipes.md [hunk 2]
```diff
@@ -1056,6 +1053,160 @@ This request is authorized for a user with the `workspaces.recipe.edit` permissi
...
+
+```shell
+curl {{ api_base_url }}/v2/recipes/{RECIPE_ID}/visibility \
+  -X PATCH \
+  -u {EMAIL}:{APIKEY} \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' -d '
+   {
+     "data": {
+       "visibility": "global"
```
Comment (HannaTrotsenko) [inline source/includes/_recipes.md]: better use contract for example

File: source/includes/_recipes.md [hunk 3]
```diff
@@ -1056,6 +1053,160 @@ This request is authorized for a user with the `workspaces.recipe.edit` permissi
...
+
+{
+  "data": {
+    "id": "{RECIPE_ID}",
+    "type": "recipe",
+    "links": {
+      "self": "/v2/recipes/{RECIPE_ID}"
+    },
+    "attributes": {
+      "visibility": "global",
```
Comment (HannaTrotsenko) [inline source/includes/_recipes.md]: better use contract for example

File: source/includes/_recipes.md [hunk 4]
```diff
@@ -1056,6 +1053,160 @@ This request is authorized for a user with the `workspaces.recipe.edit` permissi
...
+
+| Parameter | Required | Description |
+| :-------- | :------- | :---------- |
+| RECIPE_ID | yes      | Recipe ID   |
+
+### Body Parameters
+
+| Parameter                                        | Required | Description                                                        |
+| :----------------------------------------------- | :------- | :----------------------------------------------------------------- |
+| visibility                                       | yes      | Recipe sharing mode                                                |
```
Comment (HannaTrotsenko) [inline source/includes/_recipes.md]: specify in Description what variants of visibility level user can set

File: source/includes/_recipes.md [hunk 5]
```diff
@@ -6,16 +6,18 @@
...
+is available to other users for their Flows. There are four sharing modes:
 
 - ``workspace`` – Only Workspace members can use the recipe.
 - ``contract`` – Members of all Contract Workspaces can use the recipe.
 - ``tenant`` – Recipe is available to other clients in the tenant.
 - ``global`` – Any user of the platform can use these recipes.
 
-Accordingly, a set of recipes, available for each user consists of: non-shared recipes from the user's Workspace, recipes with ``contract``, ``tenant`` and ``global`` access.
-
+Accordingly, a set of recipes, available for each user consists of: non-shared recipes from the user's Workspace,
```
Comment (honeybadgeralex) [inline source/includes/_recipes.md]: Accordingly, a set of recipes available for each user consists of: non-shared recipes from the user's Workspace,

File: source/includes/_recipes.md [hunk 6]
```diff
@@ -6,16 +6,18 @@
...
+recipes with `contract`, `tenant` and `global` access. When you create recipe it has default visibility
```
Comment (honeybadgeralex) [inline source/includes/_recipes.md]: When you create a recipe, it has default visibility

**Pull request merged**